packages/lodestar-config/README.md

```diff
@@ -14,8 +14,8 @@ Configuration variables for an Eth2 network -- consists of params and ssz types
 
 ```typescript
 // mainet and minimal presets are available under non-default exports
-import {config as mainnetConfig} from "@chainsafe/lodestar-config/lib/presets/mainnet";
-import {config as minimalConfig} from "@chainsafe/lodestar-config/lib/presets/mainnet";
+import {mainnetConfig} from "@chainsafe/lodestar-config/mainnet";
+import {minimalConfig} from "@chainsafe/lodestar-config/mainnet";
```

Comment:
`import {config} from "@chainsafe/lodestar-config/mainnet";`?

Reply:
Do you suggest to only use a single import statement in the README example?

Reply:
ah I just mean the change you did everywhere is different from this README :D